docs/Installation/Linux CLI command: pt.md

```diff
@@ -0,0 +1,35 @@
+# Papertrail Linux CLI command: pt
+
+A command line interface to perform functions not found in the administrative GUI is available. This utility can be installed on Linux via pip (package management system to install Python software packages).
+
+### Requirements:
+It is not necessary for this program to be installed on the Papertrail server that you intend on connecting to. However, the following packages need to be installed on the Linux host where you intend on using the `pt` CLI command:
+* Python
+* Pip
+* Git<br>
+
+The installation of these packages may vary depending on the flavour of Linux running (e.g. *yum*, *rpm*, *apt-get*)
+
+### Installation
+Choose only one of the below options:
+
+#### Option 1
+* `sudo pip install git+ssh://git@github.com/egis/papertrail-python-cli.git`
+
+#### Installation Option 2 (should you receive an access denied message when following Option 1)
+ * Download the contents of `https://github.com/egis/papertrail-python-cli` (include sub-folders)
+ * Transfer this to the Linux host (using WinSCP as an option)
+ * Execute `sudo python setup.py install`
+
+### Usage
+ * Once installed, executing `pt` will bring up the available options and commands
```

Comment:
use pt --help to see a list of all commands available and then `pt {command} --help` for more details on the command itself e.g. `pt import --help`